### PR TITLE
Using netstandard2.0 and net60

### PR DESCRIPTION
**Pull request by mee-ironsoftware**
Note: This PR is intended to demonstrate a solution for separating net60 from netstandard2.0 and using different SixLabors.ImageSharp versions, proving that the approach works correctly.

### Title
Update IronDrawing to targets netstandard2.0 and net60 with different SixLabors.ImageSharp versions.

### Description
This PR updates IronDrawing to target both netstandard2.0 and net60 frameworks. To accommodate these changes, IronDrawing now uses different versions of SixLabors.ImageSharp for each framework: v2.1.3 for netstandard2.0 and v3.0.0 for net60. Some source code adjustments have been made to ensure compatibility with both versions of SixLabors.ImageSharp.

Fixes [#SLACK-MESSAGE](https://gettingironstuffdone.slack.com/archives/C02DANX2H35/p1678853715057679)

### Type of change
Please select the relevant option by placing an 'x' inside the brackets, like this: [x].

- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] 🏗️ Internal/structural update (non-breaking change that improves code quality, organization, or performance)
- [x] 📚 This change requires a documentation update

### How Has This Been Tested?
Run through all the Unit tests locally.

### Checklist:
Please run through the checklist as much as possible and mark the items completed by placing an 'x' inside the brackets, like this: [x].

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have successfully run all unit tests on Windows
- [x] I have successfully run all unit tests on Linux

### Additional Context
This PR prepares IronDrawing for supporting both netstandard2.0 and net60 frameworks and ensures compatibility with the respective SixLabors.ImageSharp versions.

### CODE REVIEW

**Comment by artem-iron** [comment]
Can you use this opportunity to also add netstandard 2.1 target also? This is needed to enable better mobile compatibility.

**Comment by mee-ironsoftware** [comment]
> Can you use this opportunity to also add netstandard 2.1 target also? This is needed to enable better mobile compatibility.

Good idea.

File: IronSoftware.Drawing/IronSoftware.Drawing.Common/AnyBitmap.cs [hunk 1]
```diff
@@ -641,14 +652,16 @@ public static AnyBitmap RotateFlip(AnyBitmap bitmap, RotateMode rotateMode, Flip
                 RotateMode.None => SixLabors.ImageSharp.Processing.RotateMode.None,
                 RotateMode.Rotate90 => SixLabors.ImageSharp.Processing.RotateMode.Rotate90,
                 RotateMode.Rotate180 => SixLabors.ImageSharp.Processing.RotateMode.Rotate180,
-                RotateMode.Rotate270 => SixLabors.ImageSharp.Processing.RotateMode.Rotate270
+                RotateMode.Rotate270 => SixLabors.ImageSharp.Processing.RotateMode.Rotate270,
+                _ => throw new NotImplementedException()
```
Comment (michael-ironsoftware) [inline IronSoftware.Drawing/IronSoftware.Drawing.Common/AnyBitmap.cs:656]: Good catch 🙏

**Review by michael-ironsoftware — APPROVED** [review]
LGTM

File: IronSoftware.Drawing/IronSoftware.Drawing.Common/AnyBitmap.cs [hunk 2]
```diff
@@ -478,13 +489,13 @@ public AnyBitmap(Uri Uri)
         /// <param name="Uri">The uri of the image.</param>
         /// <returns></returns>
         /// <seealso cref="AnyBitmap"/>
-        /// <seealso cref="FromUri"/>
-        public static AnyBitmap FromUri(Uri Uri)
+        /// <seealso cref="FromUriAsync"/>
```
Comment (artem-iron) [inline IronSoftware.Drawing/IronSoftware.Drawing.Common/AnyBitmap.cs:492]:
```suggestion
        /// <seealso cref="FromUri"/>
```

File: IronSoftware.Drawing/IronSoftware.Drawing.Common/AnyBitmap.cs [hunk 3]
```diff
@@ -478,13 +489,13 @@ public AnyBitmap(Uri Uri)
...
+        public static async Task<AnyBitmap> FromUriAsync(Uri Uri)
```
Comment (artem-iron) [inline IronSoftware.Drawing/IronSoftware.Drawing.Common/AnyBitmap.cs:493]: This is a breaking change, do we have any way to have the old signature still working?

**Review by artem-iron — COMMENTED** [review]
made some comments and suggestions.

File: NuGet/IronSoftware.Drawing.nuspec [hunk 4]
```diff
@@ -42,19 +42,23 @@ For general support and technical inquiries, please email us at: developers@iron
...
+      <group targetFramework="netstandard2.0">
+        <dependency id="SixLabors.ImageSharp" version="2.1.3" />
+        <dependency id="BitMiracle.LibTiff.NET" version="2.4.649" />
+      </group>
+      <group targetFramework="net60">
+        <dependency id="SixLabors.ImageSharp" version="3.0.0" />
+        <dependency id="BitMiracle.LibTiff.NET" version="2.4.649" />
+      </group>
+    </dependencies>
+
```
[the remark this answers is not in the document]
Comment (mee-ironsoftware) [reply NuGet/IronSoftware.Drawing.nuspec]: Thank you.

File: IronSoftware.Drawing/IronSoftware.Drawing.Common/AnyBitmap.cs (hunk 3 above)
Comment (mee-ironsoftware) [reply IronSoftware.Drawing/IronSoftware.Drawing.Common/AnyBitmap.cs:493]: Good catch, I'll rollback the FromUri function and mark deprecate.

**Pull request merged**